### DISCUSSION TITLE
Add a new incompressible fluid or extend the Tmin of an existing fluid in the library

### DISCUSSION THREAD

**Discussion by jblanq** [CoolProp/CoolProp#2399] [Q&A]
Hi, I would like to extend the Tmin of the NaK incompressible fluid below 290 as in https://doi.org/10.2172/46671. Or add a new fluid with this characteristic. There is a similar post of this topic, but I would like to know if there are instructions for the step-by-step procedure to do this, like the generation of the json file and compilation.

Thanks in advance!

**Comment by ibell** [comment]
You are welcome to add the instructions to the docs. CoolProp is a community project and people need to step up and get involved.
